Docs status updates - Fixes #150 (#268)
nautobot/nautobot: nautobot/docs/models/ipam/ipaddress.md

Like a prefix, an IP address can optionally be assigned to a VRF (otherwise, it will appear in the "global" table). IP addresses are automatically arranged under parent prefixes within their respective VRFs according to the IP hierarchy.

> Each IP address can also be assigned an operational status and a functional role. Default status options are provided, including the following, but may be customized as with any other "status" field in Nautobot (more information on custom status model [`here.`](https://nautobot.readthedocs.io/en/latest/models/extras/status/)):

Reviewer: ```suggestion
Each IP address can also be assigned an operational [status](https://nautobot.readthedocs.io/en/latest/models/extras/status/) and a functional role. The following statuses are available by default:
```
